src/@orb.yml

```diff
@@ -1,6 +1,12 @@
 version: 2.1
 
-description: Build, preview and publish Leanpub books
+description: >
+  Build, preview and publish Leanpub books.
+  
+  You must obtain a Leanpub API key before running this orb.
+
+  View this orb's source - https://github.com/CircleCI-Public/slack-orb
```

Comment:
Can you please fix the orb's source URL?

Reply:
Sorry for the mix-up @zzamboni , I had too many tabs open and I copied the link from the wrong one. :upside_down_face:

It should be fixed now.